cmd/list/breakglasscredential/cmd.go

```diff
@@ -0,0 +1,88 @@
+package breakglasscredential
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/externalauthprovider"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "break-glass-credentials",
+	Aliases: []string{"break-glass-credential", "breakglasscredential", "breakglasscredentials"},
+	Short:   "List break glass credential",
+	Long:    "List break glass credential for a cluster.",
+	Example: `  # List all break glass credentials for a cluster named 'mycluster'"
+  rosa list break-glass-credentials -c mycluster`,
+	Run:    run,
+	Args:   cobra.NoArgs,
+	Hidden: true,
+}
+
+func init() {
+	ocm.AddClusterFlag(Cmd)
+	output.AddFlag(Cmd)
+}
+
+func run(cmd *cobra.Command, _ []string) {
+	r := rosa.NewRuntime().WithAWS().WithOCM()
+	defer r.Cleanup()
+	err := runWithRuntime(r, cmd)
+	if err != nil {
+		r.Reporter.Errorf(err.Error())
+		os.Exit(1)
+	}
+}
+
+func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command) error {
+	clusterKey := r.GetClusterKey()
+	cluster := r.FetchCluster()
+
+	externalAuthService := externalauthprovider.NewExternalAuthService(r.OCMClient)
+	err := externalAuthService.IsExternalAuthProviderSupported(cluster, clusterKey)
+	if err != nil {
+		return err
+	}
+
+	// Load any break glass credential for this cluster
+	r.Reporter.Debugf("Loading break glass credentials for cluster '%s'", clusterKey)
+	breakGlassCredentials, err := r.OCMClient.GetBreakGlassCredentials(cluster.ID())
+	if err != nil {
+		return fmt.Errorf("failed to get break glass credentials for cluster '%s': %v", cluster.ID(), err)
```

Comment:
@chenz4027 `clusterKey` in the error message rather than `cluster.ID()` for consistency for the customer.